content/cpp/concepts/variables/terms/mutable-variables/mutable-variables.md

```diff
@@ -0,0 +1,94 @@
+---
+Title: 'Mutable variables'
+Description: 'In C++, the `mutable` keyword in C++ allows certain class variables to be modified even within const functions, useful for managing internal state flexibly.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Variables'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
```

Comment:
Modifications:

- Let's write the title in title case
- Let's change the description a bit
- Let's add some more tags

```suggestion
---
Title: 'Mutable Variables'
Description: 'Mutable variables are variables that can be modified even within constant functions.'
Subjects:
  - 'Computer Science'
  - 'Game Development'
Tags:
  - 'Functions'
  - 'Values'
  - 'Variable Types'
  - 'Variables'
CatalogContent:
  - 'learn-c-plus-plus'
  - 'paths/computer-science'
---
```